wordpress.org/public_html/wp-content/plugins/wporg-gp-translation-suggestions/templates/translation-memory-suggestions.php

```diff
@@ -23,5 +22,43 @@
 		echo '</div>';
 		echo '</li>';
 	}
+	foreach ( $openai_suggestions as $suggestion ) {
+		echo '<li>';
+		echo '<div class="translation-suggestion with-tooltip openai" tabindex="0" role="button" aria-pressed="false" aria-label="Copy translation">';
+			echo '<span class="openai-suggestion__score">OpenAI</span>';
+
+			echo '<span class="translation-suggestion__translation">';
+				echo esc_translation( $suggestion['translation'] );
+
+				if ( $suggestion['diff'] ) {
+					echo '<span class="translation-suggestion__original-diff">' . wp_kses_post( $suggestion['diff'] ) . '</span>';
+				}
+			echo '</span>';
+
+			echo '<span aria-hidden="true" class="translation-suggestion__translation-raw">' . esc_translation( $suggestion['translation'] ) . '</span>';
+
+			echo '<button type="button" class="button is-small copy-suggestion">Copy</button>';
+		echo '</div>';
+		echo '</li>';
+	}
+	foreach ( $deepl_suggestions as $suggestion ) {
+		echo '<li>';
+		echo '<div class="translation-suggestion with-tooltip deepl" tabindex="0" role="button" aria-pressed="false" aria-label="Copy translation">';
+			echo '<span class="deepl-suggestion__score">Deepl</span>';
```

Comment:
You forgot to change this one, or update it in the PR:
```suggestion
			echo '<span class="deepl-suggestion__score">DeepL</span>';
```